base/sysimg.jl

```diff
@@ -362,7 +362,7 @@ import .Random: rand, rand!
 
 # (s)printf macros
 include("printf.jl")
-using .Printf
+import .Printf
```

Comment:
Is this `import` necessary so that `Printf.@printf` overrides `Base.Printf.@printf`?